marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/endpoints/collectorLib/prepareSourceQuery.sjs

```diff
@@ -16,6 +16,8 @@ let sourceQuery = lib.prepareSourceQuery(
     type: "merging"
   }
 );
+
```

Comment:
If the code is no longer valid in collectorLib, let's just delete this and not comment it out